x/confidentialtransfers/client/cli/query.go

```diff
@@ -178,3 +186,126 @@ func queryAllAccounts(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+// GetCmdQueryTx implements a command to query a tx by it's transaction hash and return it in it's decrypted state.
+func GetCmdQueryTx() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "tx [hash]",
+		Short: "Query the confidential transaction and decrypts it",
+		Long: "Query the confidential transaction by it's tx hash and decrypts it using the private key of the account in --from" +
+			"Pass the --from flag to decrypt the account" +
```

Comment:
Passing non-owner here won't decrypt it, right? Maybe we could revise the wording everywhere a bit to be more to deliver the overall message like
``` Query for confidential transfer tx. Returns tx in decrypted form if owner addressed passed in --from flag```
We could also add example with and without `--from` flag